db/data/20240301082336_set_up_sms_reminders_bd_mar_apr2024.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+class SetUpSmsRemindersBdMarApr2024 < ActiveRecord::Migration[6.1]
+
+  EXPERIMENTS_TO_DISCARD = (3..4).map do |month_number|
```

Comment:
Since we are cancelling the experiments and not discarding (calling .discard), maybe let's call it `EXPERIMENTS_TO_CANCEL`

Reply:
> xperiments and not discarding (calling .discard), maybe let's call it `EXPERIMENTS_TO_CANCEL`

We are calling `.discard` inside the `cancel` method. That's why I named it as `EXPERIMENTS_TO_DISCARD`.